cmd/dra-example-kubeletplugin/state.go

```diff
@@ -142,14 +146,18 @@ func (s *DeviceState) Unprepare(claimUID string) error {
 	return nil
 }
 
-func (s *DeviceState) GetUpdatedSpec(inspec *nascrd.NodeAllocationStateSpec) *nascrd.NodeAllocationStateSpec {
+func (s *DeviceState) GetUpdatedSpec(inspec *nascrd.NodeAllocationStateSpec) (*nascrd.NodeAllocationStateSpec, error) {
 	s.Lock()
 	defer s.Unlock()
 
 	outspec := inspec.DeepCopy()
 	s.syncAllocatableDevicesToCRDSpec(outspec)
-	s.syncPreparedDevicesToCRDSpec(outspec)
-	return outspec
+	err := s.syncPreparedDevicesToCRDSpec(outspec)
+	if err != nil {
+		return nil, fmt.Errorf("synching prepared devices to CR spec: %v", err)
+	}
```

Comment:
Can we make the `syncAllocatableDevicesToCRDSpec()` function also return an error for symmetry in the calls here

Reply:
Done.